src/conversion.jl

```diff
@@ -28,3 +28,15 @@ for TT in (:JacobianTracer, :ConnectivityTracer, :HessianTracer)
     @eval Base.similar(::Array, ::Type{$TT{S}}, dims::Dims{N}) where {N,S} =
         zeros($TT{S}, dims)
 end
+
+Base.real(sct::SparseConnectivityTracer.AbstractTracer) = sct
+Base.float(sct::SparseConnectivityTracer.AbstractTracer) = sct
+Base.max(x::SparseConnectivityTracer.AbstractTracer, y::SparseConnectivityTracer.AbstractTracer) = x
+Base.floatmin(x::Type{<:SparseConnectivityTracer.AbstractTracer}) = Base.floatmin(Float32)
+Base.eps(x::Type{<:SparseConnectivityTracer.AbstractTracer}) = Base.eps(Float32)
```

Comment:
These should already exist:
https://github.com/adrhill/SparseConnectivityTracer.jl/blob/2fcd7308ef628c6bc3852deee75ff83b426020f9/src/operators.jl#L64-L73

Did you encounter any errors on these functions?